Frontend/src/pages/student/StudentAddNewCoop.vue

```diff
@@ -87,17 +102,31 @@ export default {
       courseTerm: "",
       courseTermOptions: ["Winter", "Summer", "Fall"],
       courseYear: new Date().getFullYear(),
-      submitting: false
+      courseName: "",
+      courseNameOptions: [],
+      submitting: false,
+      loading: true
     };
   },
+  created: function() {
+    this.$axios.get("/courses").then(resp => {
+      resp.data.forEach(course => {
+        this.courseNameOptions.push(course.name);
+        this.loading = false;
+      });
```

Comment:
i made a backend method that does this in my PR if u prefer? not sure which is better in the end but mine sends just the string names so its less bulky which is good i guess

Reply:
Oh ok nice that would be better I think since it only gets the data that's actually needed

Reply:
ok ill merge first